docker/docker-compose.yaml

```diff
@@ -84,6 +84,20 @@ services:
     ports:
       - '6379:6379'
 
+  #################################################################################
+  ##
+  ## A scalable, search-engine-boosting platform for developing models to improve
+  ## search results.
+  ##
+  #################################################################################
+  nboost:
+    image: koursaros/nboost:0.3.9-pt
+    networks:
+      - dug-network
+    command: ["--uhost", "localhost", "--uport", "9200"]
```

Comment:
This is where we'd want to add more config--I only added the bare minimum to make sure it gets up fine. Not sure what else we need to specify here? https://github.com/koursaros-ai/nboost#deploying-the-proxy